### PR TITLE
Make it clear context can be extended

### PR DESCRIPTION
**Pull request by duglin**
Signed-off-by: Doug Davis <dug@us.ibm.com>

### CODE REVIEW

**Comment by angarg12** [comment]
LGTM

**Comment by shalako** [comment]
LGTM

File: profile.md [hunk 1]
```diff
@@ -76,8 +76,9 @@ or REQUIRED level requirements defined herein.
 
 The list of properties within the Context Object can vary depending on
 which Service Broker API is being invoked and which Platform is being used.
-This section will define those properties and specify when each is meant
-to be used.
+This section will define a set of properties and specify when each is meant
```
Comment (pmorie) [inline profile.md]: set of properties for each platform

Comment (duglin) [inline profile.md]: fixed

**Comment by duglin** [comment]
another round of L G T Ms needed

**Comment by angarg12** [comment]
LGTM

**Comment by shalako** [comment]
LGTM

**Comment by pmorie** [comment]
LGTM

**Comment by zrob** [comment]
LGTM

**Pull request merged**